app/src/main/java/org/sopt/sample/util/binding/BindingFragment.kt

```diff
@@ -0,0 +1,33 @@
+package org.sopt.sample.util.binding
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+import org.sopt.sample.R
+
+abstract class BindingFragment<T : ViewDataBinding>(
+    @LayoutRes private val layoutRes: Int
+) : Fragment() {
+    private var _binding: T? = null
+    val binding get() = _binding ?: error(getString(R.string.binding_error))
```

Comment:
```suggestion
    protected val binding get() = _binding ?: error(getString(R.string.binding_error))
```
그런데.. protected 키워드를 추가해야합니당 ㅜㅜ